include/zephyr/mgmt/hawkbit.h

```diff
@@ -37,6 +37,33 @@ enum hawkbit_response {
 	HAWKBIT_PROBE_IN_PROGRESS,
 };
 
+/**
+ * @brief hawkBit event type.
+ *
+ * @details These events are used to register the callback functions.
+ *
+ */
+enum hawkbit_event_type {
+	/** Event triggered when there was an error */
+	HAWKBIT_EVENT_ERROR,
```

Comment:
It would be good if we could identify the type of error event. In our case we want to know if it was a connection/network issue.